Makefile

```diff
@@ -0,0 +1,17 @@
+
+.PHONY: test-all
+test-all: tools test-unit test-e2e
+
+.PHONY: test-unit
+test-unit:
+	@echo "Running unit tests..."
+	go test ./...
```

Comment:
Can you use here the code from the github action for the unit tests, and call the make code there?

Reply:
Once this is in place, I think we are ready to merge 😄

Reply:
done